src/components/organisms/alert-modal/index.tsx

```diff
@@ -1,43 +1,50 @@
 import { Button } from 'src/components/atoms/button/button';
 import { Card } from 'src/components/atoms/card/card';
-import { CardSlideUp } from 'src/components/templates/card-slide-up/card-slide-up';
+import { Modal } from 'src/components/templates/modal/modal';
 import { printWhen } from 'src/core/utils';
 import { AlertModalProps } from './alert-modal.types';
 import css from './alert-modal.module.scss';
 
 export const AlertModal: React.FC<AlertModalProps> = ({
   open,
   onClose,
-  header,
-  status,
   title,
+  status = '',
+  header = '',
   subtitle = '',
   footer = '',
   buttons = [],
+  contentClassName = '',
 }) => {
   return (
-    <CardSlideUp open={open} onClose={onClose}>
+    <Modal open={open} onClose={onClose} className={css.modal}>
       <>
-        <div className={css.header}>{header}</div>
+        <div className={`${header ? css.header : css.no__header}`}>
+          <span></span>
```

Comment:
Does this empty `<span>` do anything?

Reply:
a way to handle the grid to 3 sections (left, center. right) with `flex` and maybe later add something on the left side.